server/server.go

```diff
@@ -1302,16 +1303,17 @@ func (o *Options) Clone() *Options {
 
 // DefaultOptions are default options for the NATS Streaming Server
 var defaultOptions = Options{
-	ID:                DefaultClusterID,
-	DiscoverPrefix:    DefaultDiscoverPrefix,
-	StoreType:         DefaultStoreType,
-	FileStoreOpts:     stores.DefaultFileStoreOptions,
-	IOBatchSize:       DefaultIOBatchSize,
-	IOSleepTime:       DefaultIOSleepTime,
-	NATSServerURL:     "",
-	ClientHBInterval:  DefaultHeartBeatInterval,
-	ClientHBTimeout:   DefaultClientHBTimeout,
-	ClientHBFailCount: DefaultMaxFailedHeartBeats,
+	ID:                    DefaultClusterID,
+	DiscoverPrefix:        DefaultDiscoverPrefix,
+	StoreType:             DefaultStoreType,
+	FileStoreOpts:         stores.DefaultFileStoreOptions,
+	IOBatchSize:           DefaultIOBatchSize,
+	IOSleepTime:           DefaultIOSleepTime,
+	NATSServerURL:         "",
+	NATSServerCredentials: "",
```

Comment:
Not needed (since default string is empty). We could even remove `NATSServerURL` from there..